### PR TITLE
Add keys to MoveFloatToStringAndFloatTransform

### PR DESCRIPTION
**Pull request by christhetree**
to: @jq
cc: @deerzq

### CODE REVIEW

File: core/src/main/java/com/airbnb/aerosolve/core/transforms/MoveFloatToStringAndFloatTransform.java [hunk 1]
```diff
@@ -3,21 +3,23 @@
...
+ * Takes the floats in the keys of fieldName1 (or if keys are not specified, all floats) and
+ * quantizes them into buckets. If the quantized float is less than or equal to a maximum specified
+ * bucket value or greater than or equal to a minimum specified bucket value, then the quantized
+ * float is stored as a string in a new string feature output specified by stringOutputName.
+ * Otherwise, the original, unchanged float is stored in a new float feature output specified by
+ * floatOutputName. The input float feature remains unchanged.
  */
 public class MoveFloatToStringAndFloatTransform implements Transform {
   private String fieldName1;
+  private List<String> keys;
```
Comment (jq) [inline core/src/main/java/com/airbnb/aerosolve/core/transforms/MoveFloatToStringAndFloatTransform.java]: use Collection<String> keys, then below you can do if (keys == null) { keys = input.keySet()}

Comment (ozvarych) [inline core/src/main/java/com/airbnb/aerosolve/core/transforms/MoveFloatToStringAndFloatTransform.java]: Done, elegant catch!

Comment (christhetree) [inline core/src/main/java/com/airbnb/aerosolve/core/transforms/MoveFloatToStringAndFloatTransform.java]: Done, elegant catch!

**Comment by christhetree** [comment]
@jq PTAL

**Comment by jq** [comment]
👍

**Pull request merged**